test/functionalTest/cases/947_v2_entities/v2_entities_simple.test

```diff
@@ -0,0 +1,426 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
"947_v2_entities" could be more descriptive... In fact, I'd suggest to use something like:

"947_v2_list_all_entities"

That, by the way, I think is the  name of the operation in the apiary doc.